app/assets/stylesheets/components/stories.scss

```diff
@@ -133,21 +133,6 @@
     }
   }
 
-  &__flare-tag {
-    font-weight: normal;
-    font-size: 0.6em;
-    padding: 0.3em;
-    vertical-align: middle;
-    position: relative;
-    top: -2px;
-    line-height: 1em;
-    display: inline-block;
-    border-radius: var(--radius);
-    background: var(--base-60);
-    margin-right: var(--su-1);
-    color: var(--base-inverted);
-  }
-
```

Comment:
Whoops, looks like this is still needed for search results. I apologize for the confusion! Other than that, it's good to go 👍

Reply:
Added back!